trade-finance-manager-api/api-tests/v1/deals/deals-submit.facilities.api-test.js

```diff
@@ -1,9 +1,4 @@
-jest.mock('../../../src/v1/controllers/acbs.controller', () => ({
```

Comment:
Standard approach would be to add this to `beforeEach` block, where it now has been moved to.